database/transactions/transactions_test.go

```diff
@@ -0,0 +1,142 @@
+package transactions
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"sync"
+	"testing"
+	"time"
+
+	qt "github.com/frankban/quicktest"
+	dvotedb "go.vocdoni.io/dvote/db"
+	"go.vocdoni.io/dvote/db/metadb"
+	"go.vocdoni.io/dvote/util"
+)
+
+var kv TxCacheDb
+
+func TestMain(m *testing.M) {
+	storage, err := ioutil.TempDir("/tmp", ".transactions-test")
```

Comment:
```suggestion
	storage, err := ioutil.TempDir("", ".transactions-test")
```

The temp dir default is fine, and will also work on e.g. Windows.

Reply:
Even if /tmp is valid I think its better to just leave the field empty, OS will provide it.